type_specific_codegen_test.go

```diff
@@ -12,2339 +12,2339 @@ import (
   Tests for Inter (interface{} and []interface{})
 */
 func TestInter(t *testing.T) {
-  val := interface{}("something")
-  m := objx.Map{"value": val, "nothing": nil}
+	val := interface{}("something")
```

Comment:
Could you fix the indention in this file please?

Reply:
Isn't the indentation in this file more correct not vs what it was before, keeping consistent with the rest of the codebase? Even within the file itself the original indentation is inconsistent, using a full tab in some places and a half tab in others.

Reply:
Your correct, but I would rather fix this in a other PR. See #69

Reply:
I've restored the indentation